components/feature/accounts/src/main/assets/extensions/fxawebchannel/background.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*
+Establish communication with native application.
+*/
+let port = browser.runtime.connectNative("mozacWebchannelBackground");
```

Comment:
A purely stylistic suggestion, but could you please use the same constant name here as you do in the Kotlin code? Something like:

```
const WEB_CHANNEL_BACKGROUND_MESSAGING_ID = "mozacWebchannelBackground";
let port = browser.runtime.connectNative(WEB_CHANNEL_BACKGROUND_MESSAGING_ID);
```

The reason I ask is, I can see our future selves having to extend or debug something with webchannels in future, and trying to grep around in the codebase to remember where all the moving parts are. Making the different parts as similarly-shaped as possible may help that future exploration be more smooth.

Reply:
done.